js/services/RoverService.ts

```diff
@@ -0,0 +1,11 @@
+import { getRoverPhotosByDate, getRoverManifiest } from '../clients/RoverClient'
+import { UserDto } from '../dtos/UserDto'
+
+export const getRecentPhoto = async (userDto: UserDto) => {
```

Comment:
Do I need to explicitly put the return type for function?
(Ex: export const getRecentPhoto = async (userDto: UserDto): Promise)